styx-scheduler-service/src/main/java/com/spotify/styx/BackfillTriggerManager.java

```diff
@@ -158,7 +160,8 @@ boolean triggerNextPartitionAndProgress(StorageTransaction tx,
                                           String id,
                                           Workflow workflow,
                                           Instant initialNextTrigger,
-                                          int remainingCapacity) {
+                                          int remainingCapacity,
+                                          boolean reversed) {
     final Backfill momentBackfill = tx.backfill(id).orElseThrow(() ->
```

Comment:
I understand this is older, but why `momentBackfill`? Why not just backfill?

Reply:
Just to make it clear this is a fresh backfill read from database at this moment.

Reply:
It was named like this when both the `old` backfill and the one just read were in the same method, and stayed unchanged over refactorings. It is totally ok to rename if it creates confusion now.